app/models/services/pubmed/resource.rb

```diff
@@ -33,40 +22,24 @@ def map_attributes mapper, data
 
     def mapper
       {
-        title:              lambda {|data| data.xpath('//ArticleTitle').text },
-        publication:        lambda {|data| data.xpath('//Journal/Title').text },
-        doi:                lambda {|data| data.xpath('//ELocationID').text },
-        pubmed_id:          lambda {|data| data.xpath('//PMID').text },
-        abstract:           lambda {|data| data.xpath('//AbstractText').map(&:text).join("\n\n") },
-        published_at:       lambda {|data| Date.parse data.xpath('//PubDate').map(&:text).join(' ') },
-        authors_attributes: lambda {|data| authors = data.xpath('//AuthorList/Author')
+        title:              lambda {|data| data.css('ArticleTitle').text },
+        publication:        lambda {|data| data.css('Journal Title').text },
+        doi:                lambda {|data| data.css('ArticleId[IdType=doi]').text },
+        pubmed_id:          lambda {|data| data.css('PMID').text },
+        abstract:           lambda {|data| data.css('AbstractText').map(&:text).join("\n\n") },
+        published_at:       lambda {|data| Date.parse data.css('PubDate').map(&:text).join(' ') },
```

Comment:
`.css` is a convenience method that converts the css-style selector to an xpath selector. I couldn't figure out how to do `ArticleId[IdType=doi]` in xpath super easily so I converted it to `.css` instead. I figure we're already used to using CSS selectors, so why not just use it for XML?